crypto/nrf_oberon/include/ocrypto_rsa.h

```diff
@@ -191,18 +192,18 @@ int ocrypto_rsa1024_oaep_sha256_crt_decrypt(
  *
  * @param[out] s       The generated 128-byte signature.
  * @param      m       The message to be signed.
- * @param      mlen    Length of @p m.
- * @param      k       A valid 1024-bit RSA secret key.
+ * @param      m_len   Length of @p m.
+ * @param      sk      A valid 1024-bit RSA secret key.
  *
- * @return 0
+ * @retval 0
```

Comment:
```suggestion
 * @return 0 on success.
```
Using ``@retval`` only makes sense to present and explain several return values.
It's a nit and need not be changed, just saying.